proto/ojo/airdrop/v1/airdrop.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package ojo.airdrop.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/ojo-network/ojo/x/airdrop/types";
+
+option (gogoproto.goproto_getters_all) = false;
+
+// Params defines the parameters for the airdrop module.
+message Params {
+  option (gogoproto.equal) = true;
+
+  // The block at which all unclaimed AirdropAccounts will instead mint tokens
+  // into the community pool. After this block, all unclaimed airdrop accounts
+  // will no longer be able to be claimed.
+  uint64 expiry_blocks = 1;
+
+  // The percentage of the initial airdrop that users must delegate in order to
+  // receive their second portion.
+  // E.g., if we want to require users to stake their entire initial airdrop, this will be 1.
+  uint64 delegation_requirement = 2;
+  
+  // The multiplier for the amount of tokens users will receive once they claim their airdrop.
+  // E.g., if we want users to receive an equal second half, this will be 2.
+  uint64 airdrop_factor = 3;
```

Comment:
Should these be sdk.Dec instead? Realistically we'll use the values 1 and 2, but just in case there are any changes